nav2_gps_waypoint_follower_demo/launch/static_transforms.launch.py

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python3
+# Copyright (c) 2019 Samsung Research America
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+
+def generate_launch_description():
+
+    return LaunchDescription(
+        [
+            Node(
+                package="tf2_ros",
```

Comment:
Why is this not in the UDRF for the robot being provided by the robot state publisher? This is kind of hacky

Reply:
Removed, changed for urdf + robot state publisher